test/e2e/cruise/reflection/reflection_test.go

```diff
@@ -347,6 +493,16 @@ var _ = Describe("Liqo E2E", func() {
 							Name:       "secret-test-event",
 						},
 					)).To(Succeed())
+
+					// Get metrics from virtual-kubelet pod, using local portforwarding.
+					localPort := targePortMetrics // we use the same port for all providers
+					vkPod := getVirtualKubeletPod(provider.Cluster)
+					metrics := retrieveMetrics(ctx, vkPod.Name, vkPod.Namespace, localPort)
+
+					// Check the reflection counter for Ithe resource
+					counter := retrieveMetricReflectionCounter(metrics,
+						string(provider.Cluster), namespaceName, string(provider.Cluster), "Event")
+					Expect(counter).To(BeNumerically("<", 1000))
```

Comment:
Can we make this repeated piece of code a function with 2 arguments (the resource name and the limit number)?